contracts/Unibright.sol

```diff
@@ -9,24 +9,21 @@ import "@openzeppelin/contracts/access/Ownable.sol";
 
 /**
  * @title UBTSplitter
- * @dev This contract allows to split Ether payments among a group of accounts. The sender does not need to be aware
- * that the Ether will be split in this way, since it is handled transparently by the contract.
+ * @dev This contract allows to split UBT payments among a group of accounts. The sender does not need to be aware
+ * that the UBT will be split in this way, since it is handled transparently by the contract.
+ * Contract is based on PaymentSplitter, but difference is that in PaymentSplitter payees are added only once in constructor,
+ * but here can be added and update later. Because of this, contract needs to track release amount since the last payee update.
+ * Offchain solution should take care of notifying payees to pull their funds before payees are added or updated. 
  *
  * The split can be in equal parts or in any other arbitrary proportion. The way this is specified is by assigning each
- * account to a number of shares. Of all the Ether that this contract receives, each account will then be able to claim
+ * account to a number of shares. Of all the UBT that this contract receives, each account will then be able to claim
  * an amount proportional to the percentage of total shares they were assigned.
  *
  * `UBTSplitter` follows a _pull payment_ model. This means that payments are not automatically forwarded to the
  * accounts but kept in this contract, and the actual transfer is triggered as a separate step by calling the {release}
  * function.
- *
- * NOTE: This contract assumes that ERC20 tokens will behave similarly to native tokens (Ether). Rebasing tokens, and
- * tokens that apply fees during transfers, are likely to not be supported as expected. If in doubt, we encourage you
- * to run tests before sending real value to this contract.
  */
 contract UBTSplitter is Context, Ownable {
```

Comment:
Do we want to rename this to something Baseledger related? BaseledgerUbtSplitter or smt similar?